### ISSUE TITLE
Change Teach LA Green to Secondary Green

### ISSUE THREAD

**Issue by vivianha534** [uclaacm/teach-la-website#388]
Currently the site uses Teach LA's primary green color (#5EDA12) However this presents some accessibility issues. Notably, if you use a color contraster (such as this one https://webaim.org/resources/contrastchecker/) you'll see that the white against the Teach LA primary green fails. This is a big accessibility problem, and in order to fix this, we would like to change the green that's used on our site to our secondary green (#016D3A) which passes most of the tests in the color contrast.

However, note that we do not want to change Teach LA's primary green to secondary green in the case of links. For example, https://teachla.uclaacm.com/careers-cs if you go to careers in cs and hover over CS Pathways, notice how the light green expands. In these cases we want to keep the green as the Teach LA primary green.
